### DISCUSSION TITLE
Can we add support for "last day of the month" for CrontabSchedule?

### DISCUSSION THREAD

**Discussion by sanchit-mathew** [celery/django-celery-beat#685] [Ideas]
### Ask
Can we add support for "last day of the month" for `CrontabSchedule`?

### Details
We have a use-case of exporting a report, driven through a `PeriodicTask` instance with a `CrontabSchedule` schedule, on the last day of 1 or more months
This isn't currently possible and any value other than the supported values for the `day_of_month` field raise errors at the time of scheduling

### High-level Ideas
1. Add a boolean field on the `CrontabSchedule` model to represent running on the last day of the month
    - **Require** that the `day_of_month` value be set as `"28,29,30,31"`
    - Add custom scheduling logic for this in the `DatabaseScheduler`
3. Add support for a special 'L' value for the `day_of_month` 
    - Add custom scheduling logic for this in the `DatabaseScheduler`

**Comment by auvipy** [comment]
I would like to see a draft PR. but do we need Add support for a special 'L' value for the day_of_month

    Add custom scheduling logic for this in the DatabaseScheduler
 in the package or that should be project specific?

**Reply by sanchit-mathew** [reply]
@auvipy Yes, you're right, both approaches can be handled in the Crontab-specific package, not necessarily at overall project level

Thanks for the response, will try to add a draft PR soon

**Reply by gaohongsong** [reply]
Hello, we have just switched from the APScheduler component to django-celery-beat, and found that django-celery-beat does not support the 'last day of the month' attribute. Is this discussed feature still planned to be supported?

**Reply by gaohongsong** [reply]
@sanchit-mathew Excuse me, how did you implement it in the end? Could you share it?
